components/ActionSheet/tests/snapshot/MDCActionSheetControllerSnapshotTests.m

```diff
@@ -871,4 +871,55 @@ - (void)testThreeActionsSufficientSizeShortTextLTRWithDefaultPresentationStyleOn
   [self snapshotVerifyViewForIOS13:window];
 }
 
+- (void)testActionSheetWithHeaderShown {
```

Comment:
Should there be an iOS 13 dynamic color test? Will that be done in a follow-up for the theming extension?

Reply:
I don't believe so. I think we only test that dynamic colors are supported for `CGColor` instances but this is setting a UIColor on a UIView so I think we are okay.

Reply:
Acknowledged.